src/main/java/ch/ivyteam/ivy/maven/util/MavenDependencies.java

```diff
@@ -81,13 +81,9 @@ private Optional<MavenProject> findReactorProject(Artifact artifact) {
     if (session == null) {
       return Optional.empty();
     }
-    var projects = session.getProjectMap();
-    if (projects == null) {
-      return Optional.empty();
-    }
-    String artifactKey = artifact.getGroupId() + ":" + artifact.getArtifactId() + ":" + artifact.getVersion();
-    MavenProject reactorProject = projects.get(artifactKey);
-    return Optional.ofNullable(reactorProject);
+    return session.getAllProjects().stream()
```

Comment:
getAllProjects() cannot be null?

Reply:
I don't hope so. Otherwise I will add a null check :(

Reply:
I'm not sure if this is the right place for hope.

Reply:
God bless you.